deploy/awstools/awstools.py

```diff
@@ -207,12 +218,31 @@ def launch_instances(instancetype, count, instancemarket, spotinterruptionbehavi
 
         except client.exceptions.ClientError as e:
             rootLogger.info(e)
-            rootLogger.info("This probably means there was no more capacity in this availability zone. Try the next one.")
```

Comment:
I always thought the imperative 'Try the next one' was confusing in this message, like it was telling me as the user to do something instead of informing me that the script is going to try the next one.

Reply:
I agree, good catch, thanks!